docs/sections/user_guide/yaml/components/filter_topo.rst

```diff
@@ -0,0 +1,42 @@
+.. _filter_topo_yaml:
+
+filter_topo
+===========
+
+Structured YAML to run the component ``filter_topo`` is validated by JSON Schema and requires the ``filter_topo:`` block, described below. If ``filter_topo`` is to be run via a batch system, the ``platform:`` block, described :ref:`here <platform_yaml>`, is also required.
+
+Documentation for the UFS Utils ``filter_topo`` program is :ufs-utils:`here <orog-gsl>`.
+
+Here is a prototype UW YAML ``filter_topo:`` block, explained in detail below:
+
+.. highlight:: yaml
+.. literalinclude:: /shared/filter_topo.yaml
+
+UW YAML for the ``filter_topo:`` Block
+--------------------------------------
+
+config:
+^^^^^^^
+
+Configuration parameters for the ``orog_gsl`` component.
+
+  **input_grid_file:**
+
+  Path to the tiled input grid file.
```

Comment:
I nested `input_grid_file:` under a `config:` parent for consistency with the `orog_gsl` config, which has no namelist and all its other config parameters under `config:`. Generally, I think I'd find it unnecessary to have a separate section with only a single child value, but the consistency with other related drivers might be helpful.

Reply:
Because the requirements aren't generic: There's a fixed set of required files, and by being explicit about those, we can give users precise schema-checked guidance on what's required. If we found that we also needed some ability to copy generic files, we could add that, but with these tiny, old, single-purpose components, that strikes me as unlikely.